### PR TITLE
Add two new Calico Enterprise topics

### PR DESCRIPTION
**Pull request by bmckercher123**
Marketing effort to add CE content to OS documentation. 

- Add 1 new Marketing topic under Security/Calico Enterprise 
  (Advanced Egress Access Controls with Calico Enterprise)
- Added links to above doc in: Security/Adopt Zero Trust, Security/PolicyRules overview, Security/externalIPs
- Needs to merge ASAP (#1 priority for Alex/Andy)
- Written by Andy, reviewed by Alex
- Review only for broken links or misspellings

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 3a5b6196acc5456c8e86d9db741d4ba0bfeb19ed

https://deploy-preview-3170--calico.netlify.com

File: security/calico-enterprise/compliance.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Advanced compliance controls with Calico Enterprise
+description: Calico Enterprise provides complex security controls required by security teams.
+calico_enterprise: true
+---
+
+Calico policies can be used to implement security controls required by your security team. Some security controls are simple, such as separating development and test workloads or segmenting a cluster into security zones (DMZ, Trusted, etc.) or could be more complex like implementing HIPAA or PCI-DSS controls.
+
+For most security teams, setting up and enforcing Calico Policy is not sufficient to meet their needs.
+Security teams need additional controls to ensure their policies have precedence and are evaluated first before other policies
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: Missing period

File: security/calico-enterprise/compliance.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+They need to see proof that the controls are in place, being enforced, and working as expected (logs, evidence reports, audit history)
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: Missing period

File: security/calico-enterprise/compliance.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+### Flow logs
+
+- Auditors need proof that security controls are and have been enforced
+- Data in flow log
+- Screenshot
+
+### Audit logs
+
+- Marry audit log with flow logs to demonstrate adherence
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: `log` => `logs`?

File: security/calico-enterprise/compliance.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Define cluster-wide policies that cannot be overridden 
+- Tiers are evaluated left to right, top to bottom
+
+![tiered-policies]({{site.baseurl}}/images/tiered-policies.png)
+
+### Flow logs
+
+- Auditors need proof that security controls are and have been enforced
+- Data in flow log
+- Screenshot
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: Was this a placeholder for a screenshot? :)

File: security/calico-enterprise/compliance.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- Enable separation of duties
+- Define cluster-wide policies that cannot be overridden 
+- Tiers are evaluated left to right, top to bottom
+
+![tiered-policies]({{site.baseurl}}/images/tiered-policies.png)
+
+### Flow logs
+
+- Auditors need proof that security controls are and have been enforced
+- Data in flow log
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: This isn't quite a complete thought

File: security/calico-enterprise/compliance.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+### Audit logs
+
+- Marry audit log with flow logs to demonstrate adherence
+- Includes change history
+
+### Configuration auditing
+
+- CIS Benchmark Level 1 and Level 2
+- Periodically reports. Can download report from any historical point in time
+- Screenshot
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: Same comment as above -- is this just a placeholder?

File: security/calico-enterprise/compliance.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+### Evidence reports
+
+- Evidence reports show you which pods are covered by policies
+- Screenshots of reports
```
Comment (lmm) [inline security/calico-enterprise/compliance.md]: Is this a placeholder?

File: security/calico-enterprise/egress-access-control.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+title: Advanced egress access controls with Calico Enterprise
+description: Calico Enterprise provides fine-grained access control between a pod and external services outside using DNS.
+calico_enterprise: true
+---
+
+### Egress access control to external endpoints
+
+Fine-grained access control between a Pod and external service outside the cluster generally requires a firewall rule or equivalent between your Pod and the services you need to connect to. Since traditional firewall rules are defined using static IP addresses, the dynamic nature of Pod IP assignment creates challenges defining firewall rules.
```
Comment (lmm) [inline security/calico-enterprise/egress-access-control.md]: The capitalization of `Pod` is a bit unsual. I don't think we do that elsewhere in our docs.

File: security/calico-enterprise/compliance.md (hunk 1 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: Fixed.

File: security/calico-enterprise/compliance.md (hunk 4 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: Found out this morning that Compliance doc is not ready to go yet. So, yes we are not merging this doc.

File: security/calico-enterprise/compliance.md (hunk 5 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: above. not ready yet.

File: security/calico-enterprise/compliance.md (hunk 6 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: ditto comment above

File: security/calico-enterprise/compliance.md (hunk 7 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: ditto comment above

File: security/calico-enterprise/egress-access-control.md (hunk 8 above)
Comment (bmckercher123) [reply security/calico-enterprise/egress-access-control.md]: I agree. I was loathe to change anything that Andy and Alex approved in pure Marketing material. Leave as is.

File: security/calico-enterprise/compliance.md (hunk 2 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: Fixed.

File: security/calico-enterprise/compliance.md (hunk 3 above)
Comment (bmckercher123) [reply security/calico-enterprise/compliance.md]: Fixed

**Pull request merged**